crypto/src/merkle.rs

```diff
@@ -35,40 +34,40 @@ impl<T: IntoSchema> IntoSchema for MerkleTree<T> {
     }
 }
 
-/// Represents subtree rooted by the current node
+/// Represents a subtree rooted by the current node.
 #[derive(Debug)]
 pub struct Subtree<T> {
-    /// Left subtree
+    /// Left child node.
     left: Box<Node<T>>,
-    /// Right subtree
+    /// Right child node.
     right: Box<Node<T>>,
-    /// Hash of the node
+    /// Hash of this node.
     hash: HashOf<Node<T>>,
 }
 
-/// Represents leaf node
+/// Represents a leaf node.
 #[derive(Debug)]
 pub struct Leaf<T> {
-    /// Hash of the node
+    /// Hash of this node.
     hash: HashOf<T>,
 }
 
-/// Binary Tree's node with possible variants: Subtree, Leaf (with data or links to data) and Empty.
+/// Binary tree's node with possible variants: [`Subtree`], [`Leaf`] (with data or links to data) and `Empty`.
 #[derive(Debug)]
 #[allow(clippy::module_name_repetitions)]
 pub enum Node<T> {
-    /// Node is root of a subtree
+    /// [`Subtree`] node.
     Subtree(Subtree<T>),
-    /// Leaf node
+    /// [`Leaf`] node.
     Leaf(Leaf<T>),
-    /// Empty node
+    /// Empty node.
```

Comment:
```suggestion
    /// An empty node.
```

Reply:
I don't know if I should add an article to the noun at the beginning of the doc comment

Reply:
Referring back to the contributing guide, Articles are not needed if the doc string is referring to the object that it's describing, e.g. 

```
/// Node of Merkle (binary) tree
struct Node
``` 

the reasoning is to think about how it would look on docs.rs.